data/formats-data.js

```diff
@@ -5116,7 +5123,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	pawniard: {
-		randomBattleMoves: ["swordsdance", "substitute", "suckerpunch", "ironhead", "brickbreak", "knockoff"],
+		randomBattleMoves: ["swordsdance", "pursuit", "suckerpunch", "ironhead", "lowsweep", "knockoff"],
```

Comment:
brick break > low sweep
add stealth rock

Reply:
originally had stealth rock but axed it bc id rather maximize its danger, its far from a bad setter bur worse mons can take up the setting mantle
low sweep would ease  sucker mindgames but if you think bb is best im fine using it